aten/src/ATen/native/mkl/SparseCsrLinearAlgebra.cpp

```diff
@@ -7,7 +7,7 @@
 // https://github.com/pytorch/pytorch/pull/50937#issuecomment-778732740
```

Comment:
I think this comment should be removed as well? Possibly edited since the windows build fixes are implemented, but not the macos issues.